ptbcontrib/log_forwarder/requirements.txt

```diff
@@ -0,0 +1 @@
+python-telegram-bot>=20.0
```

Comment:
If you want to be on the safe side, you could use `>=20.0,<22.0`, but this is okay for me, too

Reply:
I didn't put much thought into it, I just copied from another module, the log forwarder should work with any version of `ptb` that has the same signature for `send_message`, so `>=20.0` is good.

Also, sorry for the sloppy first PR and thanks for the suggestions.